docs/section6/Rule6-9.md

```diff
@@ -0,0 +1,72 @@
+
+# Lighting - Rule 6-9
+
+**Rule ID:** 6-9  
+**Rule Description:** Baseline building is modeled with automatic shutoff controls in buildings >5000 sq.ft.  
+**Appendix G Section:** Section G3.1-6 Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:**  None  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+  1. Building total area is more than 5,000sq.ft.  
+
+**Manual Check:** Yes  
+
+**Function Call:**  
+
+  - compare_schedules()
+  - normalize_space_lighting_schedule()
+
```

Comment:
Was this function developed?

Reply:
@mkarpman not ye. Xing will work on it this week.